libs/vkd3d/resource.c

```diff
@@ -6829,13 +6829,6 @@ HRESULT vkd3d_memory_info_init(struct vkd3d_memory_info *info,
     info->cpu_accessible_domain.sampled_type_mask = sampled_type_mask_cpu & host_visible_mask;
     info->cpu_accessible_domain.rt_ds_type_mask = rt_ds_type_mask_cpu & host_visible_mask;
 
-    /* If we cannot support linear render targets, this is fine.
```

Comment:
I think info->cpu_accessible_domain for sampled and rt_ds_type_mask just need to inherit buffer_type_mask.

Reply:
Hitting `Found no suitable memory type for requested type flags #6` on NV with this PR.